packages/electron-builder/templates/nsis/boringInstaller.nsh

```diff
@@ -8,9 +8,9 @@
 !ifndef BUILD_UNINSTALLER
   Function StartApp
     ${if} ${Updated}
-      ${StdUtils.ExecShellAsUser} $0 "$startMenuLink" "open" "--updated"
+      ${StdUtils.ExecShellAsUser} $0 "$INSTDIR\${APP_EXECUTABLE_FILENAME}" "open" "--updated"
     ${else}
-      ${StdUtils.ExecShellAsUser} $0 "$startMenuLink" "open" ""
+      ${StdUtils.ExecShellAsUser} $0 "$INSTDIR\${APP_EXECUTABLE_FILENAME}" "open" ""
```

Comment:
I understand the concern, but I'm not sure whether we will face issues.
Normally, the electron application developers use app.setAppUserModelId(id) to set the AUMID.
Maybe I'm overconfident since I tested the patchset with applications that take care of setting the AUMID, but I think that this related to the owner's responsibilities.

I couldn't find a way to set the AUMID using the installer.
Unless I'm mistaken, this call System::Call 'Shell32::SetCurrentProcessExplicitAppUserModelID(w "${APP_ID}")' is meant to be used by the running process (similarly to the electron API to set the AUMID).

What do you think of the following:
 * Check whether the menu link exists
 * If yes, launch the app as it is already being done (using the shortcut).
 * If not, use the executable itself (instead of doing nothing).
Or we could add an option to let the application owner control how to launch its application (by default, start menu would be used).

It would be nice if we could also support somehow the people that make sure AUMID is set correctly for their apps.